stargate-starter/src/main/java/io/stargate/starter/Starter.java

```diff
@@ -169,13 +169,27 @@
   private boolean useProxyProtocol = false;
 
   @Order(value = 13)
+  @Option(
```

Comment:
why not add those after the last setting? To keep the ordering of those. Also, `startctl` needs to be modified.

Reply:
The goal was to group the related "proxy" settings together.

Reply:
>

Reply:
ok, I see. thx